packages/calcite-components/src/components/list-item-group/list-item-group.scss

```diff
@@ -15,20 +14,14 @@
 
 .heading {
   @apply p-0;
-  padding-inline-start: calc(
-    var(--calcite-list-item-spacing-indent) * var(--calcite-list-item-spacing-indent-multiplier)
-  );
 }
 
-::slotted(calcite-list-item) {
-  @apply shadow-border-top;
-  margin-block-start: 1px;
+::slotted(calcite-list-item:not([filter-hidden], [closed])) {
+  border-block-start: 1px solid var(--calcite-color-border-3);
 }
 
-// removes shadow for the first item of the group for both filtered and unfiltered items.
-::slotted(calcite-list-item:nth-child(1 of :not([hidden]))) {
-  @apply shadow-none;
-  margin-block-start: 0px;
+::slotted(calcite-list-item:not([filter-hidden], [closed]):nth-of-type(1)) {
```

Comment:
@ashetland because of how this selector works, applying the border to the bottom instead of the top is going to require some refactoring.

With CSS, its much easier to get the first element but getting the last one that isn't hidden by the filter or closed is kind of not doable.

We would need to refactor to remove the last item's border with JS.